translations/German/general-notes.md

```diff
@@ -0,0 +1,82 @@
+# General Rules
+
+Thank you for considering to contribute to the German translation!
+Before you start writing, please make sure that you have read the
+following general translation rules.
+
+## Formal or Informal?
+
+**Informal.**
+
+Although written German tends to be quite formal, websites in German are
+usually following informal netiquette. As Zulip's guides are written
+in a more colloquial style, German translations should be rather informal as well.
+
+**Don't use slang or regional phrases in the German translation:**
+* ~~"So'n Dreck kann jedem mal passieren."~~
+* ~~"Das ist die Seite, wo der Quelltext steht."~~
+
+## Form of address
+
+**Use "Du" instead of "Sie".**
+
+For the reasons provided in #Formal_or_Informal, stick to *Du*(informal)
+instead of *Sie*(formal) when addressing the reader.
+Always capitalize the *Du* when addressing him, do **not** use *du*.
```

Comment:
I think we could omit the second part of this sentence. 

Also, I usually use gender-neutral "them" and "they" in English when referring to a 3rd person singular of an unspecified gender - that would change to`addressing them`. It makes it easier not to make anyone feel awkward :)